src/mpl2/src/hier_rtlmp.cpp

```diff
@@ -585,16 +647,19 @@ void HierRTLMP::hierRTLMacroPlacer()
     logger_->report("The design only has macros.\n");
     hardMacroClusterMacroPlacement(root_cluster_);
   } else {
-    logger_->report(
-        "Determine shaping function for clusters -- Macro Tilings.\n");
+    debugPrint(logger_,
+               MPL,
+               "macro_placement",
+               1,
+               "Determine shaping function for clusters -- Macro Tilings.\n");
     calClusterMacroTilings(root_cluster_);
 
     // create pin blockage for IO pins
     createPinBlockage();
     //
     // Perform macro placement in a top-down manner (pre-order DFS)
     //
-    logger_->report("Perform Multilevel macro placement...");
+    logger_->report("\nPerform Multilevel macro placement...");
```

Comment:
Why does this start with \n ?

Reply:
No particular reason, but I felt easier to read the information on the log with an extra line separation this blocks